Backend/utils/seedLocation.js

```diff
@@ -0,0 +1,30 @@
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+const fs = require('fs');
+
+// Load the location data from the JSON file
+const locations = JSON.parse(fs.readFileSync('locations.json', 'utf-8'));
+
+async function main() {
+    for (const location of locations) {
+        await prisma.location.upsert({
+            where: { name: location.name },
+            update: { pathData: location.path_data },
+            create: {
+                name: location.name,
+                latitude: 0, // Use actual latitude if available
+                longitude: 0, // Use actual longitude if available
```

Comment:
what does setting latitude and longitude to 0 here do?

Reply:
when a new location is created in the database (i.e., it does not already exist), it is given that default latitude and longitude of 0, 0